packages/shared/src/components/Markdown.tsx

```diff
@@ -101,17 +92,16 @@ export default function Markdown({
         offset,
         visible: !!userId,
         onShow: cancelUserClearing,
+        onHide: clearUser,
         appendTo: appendTooltipTo || globalThis?.document?.body || 'parent',
       }}
-      onMouseEnter={cancelUserClearing}
-      onMouseLeave={clearUser}
```

Comment:
Those were passed but never used inside, just removed for clarity

Reply:
I think they where needed for the clearing right?
Now we can hover multiple users and it stays:
![Screenshot 2024-11-06 at 08 11 05](https://github.com/user-attachments/assets/d2014bea-d65b-4cde-91ce-a58ad47e1fe3)

Reply:
Nice catch, thanks for reporting! Seems that the DevCard is not closing if the mention link is the last element hovered inside the Markdown wrapper.  I'm adding a clearUser onMouseLeave in order to fix it.